metricbeat/_meta/fields.common.yml

```diff
@@ -20,6 +20,10 @@
         Connection address of the machine from which the metricset was collected. This
         field may not be present when the data was collected locally.
 
+    - name: service.hostname
+      description: >
+        Connection host name of the machine from which the metricset was collected
```

Comment:
I would try to make it clearer that this is the hostname of the machine "where the service is running".

The way it's phrased right now isn't 100% clear: "from which the metricset was collected" may mean "from which metricbeat is running", or "from which the service metrics are collected".

Otherwise, this is looking good.

Ok, I just noticed that the field above has the same ambiguity in its wording LOL. Would you mind changing both?